.gitignore

```diff
@@ -1,3 +1,4 @@
+/.fleet/
```

Comment:
Ooohh nice, didn't know you were using fleet :o

Reply:
Gave it a go, couldn't get intellisense working with PHP so it didn't last long 😅